src/orion/benchmark/assessment/paralleladvantage.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Parallel Advantage Assessment
+=============================
+"""
+
+from collections import defaultdict
+
+from orion.benchmark.assessment.base import BenchmarkAssessment
+from orion.executor.base import executor_factory
+from orion.plotting.base import durations, parallel_advantage, regrets
+
+
+class ParallelAdvantage(BenchmarkAssessment):
+    """
+    Evaluate the advantage to run experiment in multiple parallel workers.
+
+    Evaluate the average performance (objective value) for each search algorithm
+    at different time steps (trial number).
+    The performance (objective value) used for a trial will the best result until the trial.
+
+    Parameters
+    ----------
+    task_num: int
+        Number of experiment to run for each number of workers.
+    executor: str
+        Name of orion worker exeuctor.
+    n_workers: list
+        List of intergers for number of workers for each experiment.
+    executor_config: dict
+        Parameters for the corresponding executor.
+    """
+
+    def __init__(
+        self, task_num=1, executor=None, n_workers=[1, 2, 4], **executor_config
```

Comment:
```suggestion
        self, task_num=1, executor=None, n_workers=(1, 2, 4), **executor_config
```
To avoid mutable objects as default arguments.